### ISSUE TITLE
Chain not compiling 

### ISSUE THREAD

**Issue by siddharthteli12** [pendulum-chain/pendulum#327]
- Getting error **unresolved import `primitives::stellar::TransactionSetType`**,  due to `stellar-relay` in amplitude runtime. 

  https://github.com/pendulum-chain/pendulum/blob/35807d58564ab06b1a3647b7335dd81e5c43d88d/runtime/amplitude/Cargo.toml#L34

- `TransactionSetType` is define [here](https://github.com/pendulum-chain/substrate-stellar-sdk/blob/740389c7ec47a8725c32d8a68061ffb1bdfd013b/src/xdr/impls/transaction_set_type.rs#L9-L12)
- Pallet `stellar-relay` import needs to be fixed [here](https://github.com/pendulum-chain/spacewalk/blob/c2e524515fddd16c9618ff0c7c29ba2aba3f994b/pallets/stellar-relay/src/lib.rs#L40C30-L40C48)

**Comment by TorstenStueber** [comment]
@ebma can you confirm?

**Comment by siddharthteli12** [comment]
@TorstenStueber , I am not getting the above error right now. But when I am building chain with `--all-features` flag the chain is not compiling. Specifically, The error is originating from `try-runtime` feature flag. 

Apart from that I observed all previous merge's are passing GitHub actions because we are only running `cargo check` we should at least run `cargo check --all-features`. 

Will also recommend running `cargo clippy`  later after everything is sorted.

**Comment by ebma** [comment]
Yes, I can confirm that the runtimes throw errors when trying to compile them with the `try-runtime` feature. I renamed the title of this issue because that's what we need to fix. 

I'm not so sure if we should add the `cargo check --all-features` to our CI workflows though. I briefly checked the workflows of other parachain teams and it doesn't seem like they do. We might have features that are mutually exclusive and it wouldn't work to compile with all features enabled. But we can see if that is the case once the `try-runtime` compilation is fixed.

**Comment by TorstenStueber** [comment]
Yes, that's what I think. I would assume that `cargo check --all-features` is not a meaningful thing to do.

**Comment by vadaynujra** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/pendulum/327?planning-poker) with Zenhub @b-yap @bogdanS98 @ebma @gianfra-t @TorstenStueber

**Comment by gianfra-t** [comment]
This is now solved with [this](https://github.com/pendulum-chain/pendulum/pull/386) !
